components/hint-text/src/index.js

```diff
@@ -21,6 +22,13 @@ const GHintText = glamorous.span({
   },
 });
 
-const HintText = props => <GHintText {...props} />;
+const HintText = ({ children }) => (
```

Comment:
as per comment against `ErrorText`, this change should either be reverted or changed to spread `props`

(spreading `props` on `GHintText` as per the previous version of this line passes through `children` as desired, without the need to explicitly set the children)